src/pages/scripts/tutorials/tips-tricks/index.md

```diff
@@ -0,0 +1,56 @@
+# Tips and tricks for UXP scripts
+Additional tips and tricks that will help you while developing UXP Scripts.
+
+## Scripts panel
+
+The InDesign Scripts panel is the easiest and best way to run most InDesign scripts. If the panel is not already visible, you can display it by choosing Window > Utilities > Scripts.
+
+### Editing scripts
+To edit a script shown in the Scripts panel, select the script and choose Edit Script from the Scripts panel menu. Alternatively, you can hold down the Option (Mac OS) or Alt (Windows) key and double-click the script's name. This opens the script in the editor you defined for the script file type.
+
+### Open folder
+To open the folder containing a script shown in the Scripts panel, select the script and choose Reveal in Finder (Mac OS) or Reveal in Explorer (Windows). Alternatively, you can hold down the Command (Mac OS) or Ctrl-Shift (Windows) keys and double-click the script's name. The script folder opens in the Finder (Mac OS) or Explorer (Windows).
+
+### Keyboard shortcuts
+
+To add a keyboard shortcut for a script, choose Edit > Keyboard Shortcuts, select an editable shortcut set from the Set menu, then choose Product Area > Scripts. A list of the scripts in your Scripts panel appears. Select a script and assign a keyboard shortcut as you would for any other InDesign feature.
+
+
+## Run script during application startup
+
+To run a specific script when InDesign starts, put it inside a folder named "Startup Scripts" inside the Scripts folder
+- macOS - `/Applications/Adobe InDesign 2023/Scripts/startup scripts`
+- Windows - //TODO path?
```

Comment:
@anoopvalomkot  & @kareemmyk  Whats the path for Windows?

Reply:
@padmkris123 Please refer previous comment on same.

Reply:
Can't find. Can you please paste the path here?

Reply:
sorry, I misunderstood this question. The startup folder in windows is
_%APPDATA%\Adobe\InDesign\Version 19.0\en_US\Scripts\startup scripts_

Reply:
Based on your comment from another thread, do you want the users/developers to use these paths? Would the script get wiped out on new installations?

Reply:
I am attaching the screenshot from scripting guide for startup scripts.
![image](https://github.com/AdobeDocs/uxp-indesign/assets/113042047/3661e978-f21a-4440-82ae-9ecf3bface34)

Reply:
another reference on the path can be found at https://helpx.adobe.com/in/indesign/using/scripting.html

Reply:
@padmkris123 The path you have in documentation for Mac OS is pointing to install path of InDesign. 

The existing documentations (links and screen grab) that I shared above are referring to storing scripts under preferences. The attached screenshots are not for embedding in the documentation for UXP, I had taken a screen shot from the scripting guide (offline document) to share the content with you for clarification. 

I would suggest to pick the path under preferences.

Reply:
Mac : Users/[username]/Library/Preferences/Adobe InDesign/[version]/[language]/Scripts/Scripts Panel/Startup Scripts
Win : Users\[username]\AppData\Roaming\Adobe\InDesign\[version]\[language]\Scripts\Scripts Panel\Startup Scripts

The above path is sourced based on https://helpx.adobe.com/in/indesign/using/scripting.html

Reply:
@padmkris123 Startup scripts can be added under both application path and preferences path. The one under preferences will get copied when application if updated (if users chooses to copy preferences). The one under application path, in my tests, I don't see it getting copied during update, but I can't find it captured in any existing documentation.

Please stick to what is captured in existing documentation for this. I have shared the path information, in this thread and another similar thread.